CHANGELOG.md

```diff
@@ -76,7 +76,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#485](https://github.com/nf-core/sarek/pull/485) - `--skip_qc`, `--skip_markduplicates` and `--skip_bqsr` is now `--skip_tools`
 - [#538](https://github.com/nf-core/sarek/pull/538) - `--sequencing_center` is now `--seq_center`
-- [#538](https://github.com/nf-core/sarek/pull/538) - `--markdup_java_options` has been removed
```

Comment:
did you keep it now afterall?

Reply:
no, I must have commited the wrong version of `CHANGELOG`